CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file.
 
+## [7.1.0](https://github.com/voxpupuli/beaker-rspec/tree/7.1.0) (2022-01-14)
+
+[Full Changelog](https://github.com/voxpupuli/beaker-rspec/compare/7.0.0...7.1.0)
+
+**Implemented enhancements:**
+
+- Reduce beaker-pe to a development dependendency [\#109](https://github.com/voxpupuli/beaker-rspec/pull/109) ([ekohl](https://github.com/ekohl))
+
+**Merged pull requests:**
+
+- Add acceptance tests to CI [\#110](https://github.com/voxpupuli/beaker-rspec/pull/110) ([ekohl](https://github.com/ekohl))
```

Comment:
For clarity, should this also mention beaker-pe was dropped altogether?

Reply:
good idea, I will update the PR title

Reply:
updated it. do you think this change quallifies for a major release? (I don't think so)

Reply:
I don't. It's easy enough for people to pull in if they need it.